playbooks/ci-ceph_swift/templates/usr/bin/lshw

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+hostname="{{ inventory_hostname }}"
+
+{% raw -%}
+# run the original lshw command
+if [ "$1" != "-C" -o "$2" != "disk" ]; then
+  if [ -f /usr/bin/lshw.original ]; then
+    sudo /usr/bin/lshw.original
```

Comment:
Would this even work? That path isn't in the sudoers configuration. But isn't this script itself called with sudo? Would you need to re-sudo here?